packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,79 @@
+.modal {
+    position: absolute;
+    background: var(--bg-color);
+    z-index: 1000;
+    left: 50%;
+    top: 50%;
+    width: 50%; /* 6 cols = 50% */
+    transform: translate(-50%, -50%);
+    border-radius: 4px;
+    padding: 32px;
+}
+
+.modal-root {
+    display: block;
+}
+
+.modal__open-true {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr); 
+    visibility: visible;
+}
+
+.modal__open-false {
+    display: none;
+    visibility: hidden;
+}
+
+.modal-content {
+    grid-row: 3 / auto;
+    grid-column: 1 / 12;
+}
+
+.modal__wrapper-true {
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(89, 89, 89, 0.7);
+}
+
+.modal__wrapper-false {
+    display: none;
+}
+
+.modal-close {
+    background: transparent;
+    display: flex;
+    grid-row: 1;
+    grid-column: 12;
+    justify-content: center;
+    align-items: center;
+    justify-self: end;
+}
+
+.modal-close:hover {
+    cursor: pointer;
+}
+
+.modal .modal-title {
+    grid-row: 2;
+    grid-column: 1 / 12;
+    font-size: 24px;
+    font-weight: 500;
+    font-style: normal;
+    line-height: 28.8px;
```

Comment:
Lets allow the user to decide the line-height (we can add a CSS var for this later, for now you can remove this line)